wazo_call_logd/plugins/cdr/schemas.py

```diff
@@ -61,6 +61,17 @@ class RecordingMediaExportSchema(Schema):
     uuid = fields.UUID()
 
 
+class RemotePartySchema(Schema):
+    kind = fields.String(
+        validate=OneOf(['user', 'conference', 'meeting']), missing='unknown'
+    )
+    description = fields.Dict(
+        keys=fields.String(),
+        values=fields.String(),
+        required=False,
```

Comment:
`required=True` ?

Either the description is empty {} or it contains the id+name.

Reply:
no, it is not required; only the `type` is required